org.elysium/src/org/elysium/resource/LilyPondResourceDescriptionStrategy.java

```diff
@@ -19,24 +18,28 @@
 public class LilyPondResourceDescriptionStrategy extends DefaultResourceDescriptionStrategy {
 
 	@Override
+	//only root assignmnets are visible to the outside
+	//TODO: please confirm that index is not needed for click and point; for me it worked without the user data!
```

Comment:
I can confirm that point-and-click works indeed. The rationale behind the old implementation was to avoid unnecessary automatic rebuilds; however, since the point-and-click coordinates (line and offset-in-line, not even the offset-in-file as in the old implementation) have to be included in it, I don't think that really helped in many cases. Maybe the hash should include a list of the coordinates of all Pitch and Assignment children?